recipes/pyvmomi/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyvmomi" %}
+{% set version = "6.7.1.2018.12" %}
+
+ package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 653f978fd14474a1d3ffc22b1fdffa8a406987ada4ef6d13d96a37ef4b038891
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

`--no-deps --ignore-installed` are included by default
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```